rllib/tests/backward_compat/checkpoints/v1.0/dqn_frozenlake_tf/policies/default_policy/rllib_checkpoint.json

```diff
@@ -1 +1 @@
-{"type": "Policy", "checkpoint_version": "1.0", "ray_version": "3.0.0.dev0", "ray_commit": "5ae1638c8797518dc2177d7cf5d327cd9c32d706"}
\ No newline at end of file
+{"type": "Policy", "checkpoint_version": "1.0", "ray_version": "2.8.0", "ray_commit": "5ae1638c8797518dc2177d7cf5d327cd9c32d706"}
```

Comment:
I doubt these checkpoints require changing, but in the past people always change them.. so probably does not really matter.

I would revert them. it has a `ray_commit`, which won't match anyways after the version change.

I think these are just data files (from example but real runs) for some form of testing.